spec.html

```diff
@@ -13393,6 +13417,33 @@ <h2>Supplemental Syntax</h2>
     <emu-clause id="sec-static-semantics">
       <h1>Static Semantics</h1>
 
+      <emu-clause id="sec-left-hand-side-expressions-static-semantics-early-errors">
+        <h1>Static Semantics: Early Errors</h1>
+        <emu-grammar>
+          OptionalChain[Yield, Await] :
+            `?.` TemplateLiteral[?Yield, ?Await, +Tagged]
+            OptionalChain[?Yield, ?Await] TemplateLiteral[?Yield, ?Await, +Tagged]
+        </emu-grammar>
+        <ul>
+          <li>
+            It is a Syntax Error if any code matches this production.
+          </li>
+        </ul>
+        <emu-note>
+          <p>This production exists in order to prevent automatic semicolon insertion rules (<emu-xref href="#sec-automatic-semicolon-insertion"></emu-xref>) to be applied to the following code:</p>
```

Comment:
```suggestion
          <p>This production exists in order to prevent automatic semicolon insertion rules (<emu-xref href="#sec-automatic-semicolon-insertion"></emu-xref>) from applying to the following code:</p>
```